spec/unit/enforcement_spec.rb

```diff
@@ -19,10 +33,20 @@
       solr_parameters[:fq].first
     end
 
+    # rubocop:disable RSpec/MessageExpectation
+    describe 'logger' do
+      let(:ability) { instance_double(Ability, user_groups: [], current_user: user) } # prevent Ability from hitting logger first
```

Comment:
The expectation set will be falsely triggered by Ability class (that also calls Rails.logger.debug frequently).  So we use a double'd version of Ability to avoid tripping that code.